src/controllers/document_upload/ajax.upload.responder.strategy.ts

```diff
@@ -50,9 +50,10 @@ export class AjaxUploadResponderStrategy implements UploadResponderStrategy {
    * @param {Error} e the error
    * @param {NextFunction} _next the next middleware function
    */
-  public handleGenericError = (res: Response, e: Error, _next?: NextFunction) => {
+  public handleGenericError = (res: Response, e: Error, _next?: NextFunction): Promise<void> => {
     logger.error(ErrorMessages.ERROR_500 + ": " + e);
     res.status(500).send({ redirect: pageURLs.OBJECTIONS_ERROR });
+    return Promise.resolve();
```

Comment:
Why do we need these? Is this something es lint has complained about or are there problems that have resulted from the absence of this?